### ISSUE TITLE
How to ignore only "position" warnings?

### ISSUE THREAD

**Issue by Lamecarlate** [efficientcss/stylelint-config-ecss#1]
Hello!

I discovered ECSS some weeks ago and plunged in it. Been redesigning my website, so it was just in time! Great stylelint config too.

However I have difficulties to ignore some rules in my .stylelintrc.json. I would like to **ignore the warnings** on "declaration-property-value-disallowed-list" but **only on "position" keyword**. I didn't see any other warnings for now (good news) and would like to avoid disabling the entire rule. How do I write this?

I tried: 

```
"declaration-property-value-disallowed-list": [
          [{ "keywordId": "position-sensitive", "severity": null }]
]
```

and some other combinations I'm a bit ashamed of. The warnings have disappeared but I fear this is a false negative, because if I write a non-existent keywordId, the warnings don't reappear.

Please help me :)

**Comment by emmacharp** [comment]
HI @Lamecarlate !

Great to hear you find it useful. Could you share your "faulty" code so I can better understand a possible solution? Because as it is, there is no way of disabling through the json file.

If we can't resolve your specific problem, I'll give you some alternatives.

**Comment by Lamecarlate** [comment]
For example, this one:

```
.theme-switcher {
  position: relative;

  [id="theme-switcher-list"] {
    position: absolute;
  }
}
```

This fires a warning: 

> Cette valeur de positionnement est délicate. Prudence est de mise.

(by the way I love the translation, it's really neat)

I know absolute is tricky, and I'm glad that there are warnings about that. But I've been a webdeveloper for more than ten years, I thinks I can manage this properly 🙂 and I don't really need the warning.

If it can't be hidden, so be it, I'll endure 😄

**Comment by emmacharp** [comment]
I see! I'd suggest you disable warnings by using `-q` in your stylelint command. That'd be the simplest way to go about it I think.

Keep me posted.

PS : content que la traduction vous serve !

**Comment by Lamecarlate** [comment]
I did not know about the `-q` param. Thanks for that! I'll use that, and duplicate my custom command, to do a little complete overhaul, warnings included, from time to time.

Do you know if there will be a way to disable precise warnings in the future, or is this impossible due to the nature of stylelint?

**Comment by emmacharp** [comment]
The thing is I use one Stylelint rule for multiple warnings. You could odisable all warnings associated to this rule, but not only the `position` one. 

`"declaration-property-value-disallowed-list": null` should do the trick if you're willing to disable all of them (you can look at the `index.js`file to see which warnings are included in the rule).

I'm considering creating one rule per ECSS error/warning exactly for the kind of control you'd like to have... but I'm still not sure if the amount of code necessary for it would be justified. And I'm kind of lazy. Heh.

**Comment by emmacharp** [comment]
Closing. I'll reopen if you have any problem.
